mlrun/artifacts/base.py

```diff
@@ -798,16 +863,27 @@ def spec(self, spec):
     def is_dir(self):
         return True
 
-    def upload(self):
+    def upload(self, artifact_path: str = None):
+        """
+        internal, upload to target store
+        :param artifact_path: required only for when generating target_path from artifact hash
+        """
         if not self.spec.src_path:
             raise ValueError("local/source path not specified")
 
         files = os.listdir(self.spec.src_path)
-        for f in files:
-            file_path = os.path.join(self.spec.src_path, f)
+        for file in files:
```

Comment:
It's best practice to not use the variable name `file` as it's already a type/class in python. I suggest changing it back to `f` or `artifact_file`

Reply:
changed to `file_name`